src/Resources/public/js/workspace/abstract.js

```diff
@@ -34,12 +35,14 @@ pimcore.plugin.datahub.workspace.abstract = Class.create({
         ];
 
         var check;
-        for (var i=0; i<availableRights.length; i++) {
+        for (var i=0; i<this.availableRights.length; i++) {
 
             var checkConfig = {
-                text: t("plugin_pimcore_datahub_workspace_permission_" + availableRights[i]),
-                dataIndex: availableRights[i],
-                width: 70
+                text: t("plugin_pimcore_datahub_workspace_permission_" + this.availableRights[i]),
+                dataIndex: this.availableRights[i],
+                width: 70,
+                hidden: this.rightCheckboxHidden || false,
```

Comment:
?

Reply:
To hide the column. In my use case i have only the "read" permission which is allways checked and it should be hidden by default - and disabled...